src/annotation/index.ts

```diff
@@ -30,6 +30,63 @@ export class Annotations {
       }) - 1;
   };
 
+  deleteActiveAnnotation = (): void => {
+    this.data.splice(this.activeAnnotationID, 1);
+    if (this.activeAnnotationID >= this.data.length) {
+      this.activeAnnotationID = this.data.length - 1; // necessary if we delete the one on the end
+    }
+    if (this.data.length === 0) {
+      this.addAnnotation("paintbrush"); // re-create a new empty annotation if we delete the last one (toolbox will be re-assigned by reuseEmptyAnnotation if necessary)
+    }
+  };
+
+  getLabels = (): string[] => this.data[this.activeAnnotationID].labels;
```

Comment:
Should we use CRUD terminology here? I.e. createLabels, readLabels, updateLabels, deleteLabels?

Reply:
We currently have: delete, get, add, clear, update, insert, and set

Reply:
I tend to use get/create fwiw

Reply:
![image](https://user-images.githubusercontent.com/3502798/119374393-6edad300-bcb1-11eb-824a-f46bcc6187d5.png)